### ISSUE TITLE
Can you provide the data set?

### ISSUE THREAD

**Issue by kushal-chakraborty** [dongjk/faster_rcnn_keras#5]

**Comment by dongjk** [comment]
1，download data https://www.kaggle.com/c/imagenet-object-detection-challenge/data
2，unzip it to your path
3，modify code, change ILSVRC_dataset_path to your path

**Comment by kushal-chakraborty** [comment]
The link does not seem to work. It says the page cannot be opened. Actually I'm trying to extract region proposals of an image. Can you please help me? I am a 2nd year undergraduate and a newbie in this area.

The code I tried earlier:

k=9 
feature_map_tile = Input(shape=(None,None,3))
convolution_3x3 = Conv2D(filters=512,kernel_size=(3, 3),padding='same',name="3x3")(feature_map_tile)

output_deltas = Conv2D(filters= 4 * k,kernel_size=(1, 1),activation="linear",kernel_initializer="uniform",name="deltas1")(convolution_3x3)

output_scores = Conv2D(filters=1 * k,kernel_size=(1, 1),activation="sigmoid",kernel_initializer="uniform",name="scores1")(convolution_3x3)

model = Model(inputs=[feature_map_tile], outputs=[output_scores, output_deltas])
model.compile(optimizer='adam', loss= 'mean_squared_logarithmic_error')

model.predict(img)

but it does not produce region proposals
The output I get is given in the link:
https://drive.google.com/file/d/1nOCktVabx075k_MJ5Yits7fe55_phSM_/view?usp=sharing
